toggles/webapp/toggles.ts

```diff
@@ -58,6 +58,13 @@ const toggles = {
       description:
         "Has stories search use the Content API instead of Prismic's API.",
     },
+    {
+      id: 'useIiifTest',
```

Comment:
I'm not unreasonable enough to suggest that you change it, but this capitalisation is troubling and I think `useIIIFTest` would be slightly more consistent

Reply:
done